src/main/java/de/numcodex/sq2cql/model/cql/FunctionInvocation.java

```diff
@@ -0,0 +1,29 @@
+package de.numcodex.sq2cql.model.cql;
+
+import static java.util.Objects.requireNonNull;
+
+import de.numcodex.sq2cql.PrintContext;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public record FunctionInvocation(String identifier, List<Expression> paramList) implements
+    InvocationTerm {
+
+
+  public FunctionInvocation {
+    requireNonNull(identifier);
+    paramList = List.copyOf(paramList);
+  }
+
+  public static FunctionInvocation of(String identifier, List<Expression> paramList) {
+    return new FunctionInvocation(identifier, paramList);
+  }
+
+  @Override
+  public String print(PrintContext printContext) {
+    printContext.resetPrecedence();
+    return "%s(%s)".formatted(identifier,
```

Comment:
Can be:
```java
"%s(%s)".formatted(identifier, paramList.stream().map(printContext::print).collect(joining(", ")))
```
See also ListSelector.
We should definitely have a spec after the comma.